src/routes/Dashboard/Dashboard.svelte

```diff
@@ -77,15 +82,15 @@
           // This is a hack, but the speedometer plugin doesn’t give any
           // way to customize these labels.
           setTimeout(() => {
```

Comment:
I think you can remove this entire setTimeout hack to set the label. AQI values can’t be larger than 500, so there’s no need to change "500" --> "500+".

Reply:
Good call! I didn't realize 500 was our ceiling